crates/ev-cli/src/commands/interact.rs

```diff
@@ -0,0 +1,201 @@
+use crate::theme::CliTheme;
```

Comment:
this file is lifted almost directly - updated the validators to all return results instead of some of them returning is_valid bools.